grantnav/csv_layout.py

```diff
@@ -134,3 +153,35 @@
     "max",
     "min"
 ]
+
+
+def grants_csv_to_dictionary():
+    """ takes grants_csv and turns it into a dictionary of parent/child
+    fields e.g.
+
+    res["Planned Dates"][
+     {'title': 'Start Date', 'path': 'result.plannedDates.0.startDateDateOnly'},
+     {'title': 'End Date', 'path': 'result.plannedDates.0.endDateDateOnly'},
+     {'title': 'Duration (months)', 'path': 'result.plannedDates.0.duration'}
+    ]
+    """
+    res = {}
+
+    for title, path in grants_csv.items():
+        parsed = title.split(":")
```

Comment:
Is it possible to have more than one `:` in a title?
```suggestion
        parsed = title.split(":", maxsplit=1)
```